pig-tests/src/org/partiql/pig/tests/generated/sample-universe.kt

```diff
@@ -7318,10 +7318,12 @@ class DomainA private constructor() {
          */
         fun productA(
             one: Long,
+            two: ProductToRemove,
```

Comment:
Unrelated to this change, but the formatting of permuted domain visitor transforms is slightly off:

```
open fun transformUnpermutedProduct(node: DomainA.UnpermutedProduct): DomainB.UnpermutedProduct {
        val new_foo = transformUnpermutedProduct_foo(node)
        val new_bar = transformUnpermutedProduct_bar(node)
        val new_metas = transformUnpermutedProduct_metas(node)
        return             DomainB.UnpermutedProduct(
                foo = new_foo,
                bar = new_bar,
                metas = new_metas
            )
    }
```

Reply:
That's been there for a while.  It's also not that important, IMHO, to get generated code to be perfectly formatted.